### DISCUSSION TITLE
How do you change the default indent size for R?

### DISCUSSION THREAD

**Discussion by TimTaylor** [posit-dev/positron#5273] [Q&A] [answered]
Is it possible to change the default indent size for R?

In https://github.com/posit-dev/positron/issues/3691#issuecomment-2201108873 it says
> 
> You can configure the tab size for R specifically to override this config we set based on norms in the R community, but we can gather more feedback on whether we should change the default behavior.

but I can't seem to work out how to change it. I've tried the following in `settings.json` to no avail but I admit I'm quite ignorant to whether that should have worked or not

```json
{
    "editor.fontSize": 16,
    "debug.console.fontSize": 16,
    "terminal.integrated.fontSize": 16,
    "markdown.preview.fontSize": 16,
    "editor.fontFamily": "Fira Code",
    "editor.fontLigatures": true,
    "workbench.reduceMotion": "on",
    "editor.tabCompletion": "on",
    "files.trimTrailingWhitespace": true,
    "editor.tabSize": 4,

    "[R]": {
        "editor.tabSize": 4
    },
}
```

**Comment by TimTaylor** [comment, marked as answer]
Need to use `"[r]":  {tabSize": 4}` not `"[R]":  {tabSize": 4}`.  Arrrrrrrrggghhh :sweat_smile:

**Reply by dbcoffee** [reply]
I've run into the same problem. Could you please guide me on how to adjust the settings to resolve this?😅
